node/pkg/checker/dal/app.go

```
@@ -147,6 +148,16 @@ func checkDal(endpoint string, key string, alarmCount map[string]int) error {
 	)
 	networkDelay := time.Since(now)
 
+	if networkDelay > NetworkDelayThreshold {
+		networkDelayAlarmCount++
+		if networkDelayAlarmCount > AlarmOffsetInTotal {
+			msg += fmt.Sprintf("(DAL) network delay: %s\n", networkDelay)
+			networkDelayAlarmCount = 0
```

golangci-lint (1.62.2)

ineffectual assignment to networkDelayAlarmCount

(ineffassign)